xc/epoll.ml

```diff
@@ -0,0 +1,24 @@
+open Unix
```

Comment:
Do we still need this and the stubs? I thought we were using core

Reply:
Oh, sorry, I made a mistake, removed this file.